tmp/build/patch-jaeger.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+function jaeger_patch_values() {
+	# update jaeger zipkin port name
+  sed -i -e '/service:$/,/externalPort:/ {
```

Comment:
@rcernich No idea what this is doing. Doesn't look like it gets reflected in the final values.yaml. Reverted for now.